e2e/src/test/java/org/jboss/sbomer/test/e2e/E2EBase.java

```diff
@@ -167,6 +205,23 @@ public void publishedUmbMessage(String generationRequestId, Consumer<Validatable
         consumer.accept(atomicResponse.get());
     }
 
+    public Response getRequest(String requestId) {
+        log.info("Fetching request with id '{}'", requestId);
+
+        Response response = RestAssured.given()
+                .baseUri(getSbomerBaseUri())
+                .contentType(ContentType.JSON)
+                .pathParam("id", requestId)
+                .log()
+                .all()
+                .when()
+                .get("/api/v1beta1/requests/id={id}");
```

Comment:
@dwalluck had a question about this line, it also kinda caught me out as the other endpoints simply take the id without a named parameter required, its also not a query param like we see elsewhere. 

The syntax I think is correct though based off examples in https://stackoverflow.com/questions/32475850/how-to-pass-parameters-to-rest-assured

Also worth pointing out I added raw request logging as I'd hit a few problems (pathParam formatting being one), i figured it would be helpful to leave on